Audacia.Quest/Audacia.Quest.Core/Time.cs

```diff
@@ -1,7 +1,21 @@
-﻿namespace Audacia.Quest.Core
+﻿using System.Diagnostics;
+
+namespace Audacia.Quest.Core
 {
     public class Time
     {
-        public static float DeltaTime { get; set; }
+        private readonly static Stopwatch _stopwatch = new Stopwatch();
+
+        public static void Start()
+        {
+            _stopwatch.Reset();
+            _stopwatch.Start();
+        }
+
+        public static void Update()
```

Comment:
This update function is empty
1. if it's intended for overloading then make it virtual
2. if it's not going to be used at all then remove it
3. otherwise if you have plans for later this is totally fine

Reply:
I plan on using it in the future